### PR TITLE
Fix bead name types

### PR DESCRIPTION
**Pull request by JoshuaSBrown**

### CODE REVIEW

**Comment by JoshuaSBrown** [comment]
Don't merge yet!

**Comment by JoshuaSBrown** [comment]
Ok good to merge.

File: src/libcsg/modules/io/lammpsdatareader.cc [hunk 1]
```diff
@@ -277,7 +279,8 @@ void LAMMPSDataReader::InitializeAtomAndBeadTypes_() {
   }
 
   auto baseNamesMasses = determineBaseNameAssociatedWithMass_();
-  auto baseNamesCount = determineAtomAndBeadCountBasedOnMass_(baseNamesMasses);
+  // auto baseNamesCount =
+  // determineAtomAndBeadCountBasedOnMass_(baseNamesMasses);
```
Comment (junghans) [inline src/libcsg/modules/io/lammpsdatareader.cc]: Remove

Comment (JensWehner) [reply src/libcsg/modules/io/lammpsdatareader.cc]: please

**Review by JensWehner — APPROVED** [review]

**Pull request merged**